moz-fx-data-shared-prod/org_mozilla_firefox/event_types_v1/metadata.yaml

```diff
@@ -9,6 +9,6 @@ labels:
 scheduling:
   dag_name: bqetl_fenix_event_rollup
   destination_table: null
-  sql_file_path: 'sql/org_mozilla_firefox/event_types_v1/query.sql'
-  referenced_tables: [['org_mozilla_firefox_derived', 'event_types_v1']]
+  sql_file_path: 'sql/moz-fx-data-shared-prod/org_mozilla_firefox/event_types_v1/query.sql'
+  referenced_tables: [['moz-fx-data-shared-prod', 'org_mozilla_firefox_derived', 'event_types_v1']]
```

Comment:
Why do these `referenced_tables` lists now need to include project? Was `moz-fx-data-shared-prod` previously hard-coded in the dag generation logic?

Reply:
Yes, it was hardcoded into the template before.